Jenkinsfile_nightly

```diff
@@ -0,0 +1,17 @@
+#!groovy
```

Comment:
This PR includes #104 given suppressions required from there too.

Reply:
okokokkokokoi